features/step_definitions/sriov.rb

```diff
@@ -0,0 +1,122 @@
+Given /^the sriov operator is running well$/ do
+  ensure_destructive_tagged
+  ensure_admin_tagged
+  step %Q/I switch to cluster admin pseudo user/
+  step %Q/evaluation of `cluster_version('version').channel.split('-')[1]` is stored in the :ocp_cluster_version clipboard/
+
+  unless project('openshift-sriov-network-operator').exists?
+    if cb.ocp_cluster_version.include? "4.[34]."
+      # create 4.3/4.4 namespaces
+      namespace_yaml ||= "#{ENV['BUSHSLICER_HOME']}/testdata/networking/sriov/namespace/43/ns.yaml"
+      @result = admin.cli_exec(:create, f: namespace_yaml)
+      raise "Error creating namespace for openshift-sriov-network-operator" unless @result[:success]
+    else
+      # Create namespace for 4.5 and later
+      namespace45_yaml ||= "#{ENV['BUSHSLICER_HOME']}/testdata/networking/sriov/namespace/45/ns45.yaml"
+      @result = admin.cli_exec(:create, f: namespace45_yaml)
+      raise "Error creating namespace for openshift-sriov-network-operator" unless @result[:success]
+    end
+    
+  end
+  step %Q/I use the "openshift-sriov-network-operator" project/
+  unless operator_group('sriov-network-operators').exists?
+    og_yaml ||= "#{ENV['BUSHSLICER_HOME']}/testdata/networking/sriov/og/og.yaml"
+    @result = admin.cli_exec(:create, f: og_yaml)
+    raise "Error creating og for openshift-sriov-network-operator" unless @result[:success]
+  end
+  
+  unless subscription('sriov-network-operator-subsription').exists?
+     sub_yaml ||= "#{ENV['BUSHSLICER_HOME']}/testdata/networking/sriov/subscription/sub.yaml"
+     step %Q/I process and create:/, table(%{
+          | f | #{sub_yaml}                        |
+          | p | CHANNEL=#{cb.ocp_cluster_version}  |
+     })
+     raise "Error creating subscription for sriov operator" unless @result[:success]
+  end
+  
+  step %Q/sriov operator is ready/
+  step %Q/sriov config daemon is ready/
+end
+
+Given /^sriov operator is ready$/ do
+  ensure_admin_tagged
+  project("openshift-sriov-network-operator")
+  step %Q/a pod becomes ready with labels:/, table(%{
+    | name=sriov-network-operator |
+  })
+end
+
+Given /^sriov config daemon is ready$/ do
+  ensure_admin_tagged
+  project("openshift-sriov-network-operator")
+  step %Q/a pod becomes ready with labels:/, table(%{
+    | app=sriov-network-config-daemon |
+  })
+end
+
+Given /^I create resource with following:$/ do | table |
```

Comment:
@zhaozhanqi this step looks like is `sriov` speciifc?  If that's the case, we should change the step name to indicate that. `I create resource` is too generic.

Reply:
thanks your advice @pruan-rht , how about `I create sriov resource with following`